### ISSUE TITLE
The storage submodule should allow public access prevention

### ISSUE THREAD

**Issue by KevinBrooke** [terraform-google-modules/terraform-google-log-export#169]
### TL;DR

modules/storage/main.tf
Add the option for public_access_prevention

### Terraform Resources

```markdown
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/storage_bucket#public_access_prevention
```

### Detailed design

```markdown
CIS 5.1 requirement - Ensure that Cloud Storage bucket is not anonymously or publicly accessible.
```

**Comment by KevinBrooke** [comment]
Using another module just for the storage account, but it would be nice to have this module do it.

**Comment by imrannayer** [comment]
fixed in #181
